src/Controller.cpp

```diff
@@ -141,15 +141,15 @@ void Controller::process(byte num_messages)
     // at least one CAN frame is available in the reception buffer
     // retrieve the next one
 
-    _msg = transport->getNextMessage();
+    CANFrame msg = transport->getNextMessage();
```

Comment:
Making the message a local variable instead of a class field. It is only used here.